_articles/bluetooth.md

```diff
@@ -13,8 +13,68 @@ section: network-troubleshooting
 
 ---
 
+# Important Notes About Bluetooth
+
+Bluetooth is a bit odd. 
+There are a lot of factors that go into whether Bluetooth devices work as expected.
+
+
+## Bluetooth version:
+
+Bluetooth 5.0 is backwards compatible with older Bluetooth versions but, older bluetooth versioned devices are not always compatible with newer devices.
+
+
+## Signal Interference:
+
+Bluetooth uses the same bandwidth as the 2.4Ghz Wi-Fi band, and in most of our machines it is on the same chip as the Wi-Fi module. They usually have two antennae, one for Bluetooth, and one for Wi-Fi, but it is possible for other Wi-Fi or Bluetooth devices signals to cross and to cause connection issues.
+
+
+## Device Specific Differences:
+
+Every Bluetooth device is different. They use the same or similar protocols, but the printed circuit boards (PCBs), are specific to each device, and the firmware they are running is often custom designed, and closed-source.
+
+Some of the code and technologies that make Bluetooth work reliably are patented, and only device vendors who have licensed the permission to use that patented technology will be able to experience the full benefits (see "audio input/output" section below.)
+
+
+## Kernel Versions
+
+As the Linux kernel develops, support for more devices are added. Sometimes Bluetooth devices will work better in a different kernel version.
+
+
+## OS Versions
+
+Similar to the kernel versions. Improvements are often made in newer versions of Ubuntu and PopOS. Running software updates is always a good idea, followed by a reboot.
+
+<!--## Bluez Versions-->
+
+## Configuration Issues:
+
+Sometimes Bluetooth devices are working correctly, but something in settings needs to be reset.
+
+The easiest way to test this is to "forget" the paired Bluetooth device, and pair it again.
+
+A more thorough way of testing this would be to create a [test user](https://support.system76.com/articles/other-accounts/), or boot from a [Live Disk](https://support.system76.com/articles/live-disk/) to see if Bluetooth works in either case. 
+If it does, config files may need deleted. If it doesn't (especially in the Live Disk), reinstalling the OS may solve the problem. 
+Reinstalling the OS won't affect Bluetooth hardware directly, but resetting and starting with a clean slate can solve a slew of problems and save time hunting for a specific file or bug.
+
+
+# Setting Expectations:
+
+Because of all of these factors, if the steps outlined in the Bluetooth troubleshooting article, and the previous troubleshooting steps don't resolve the issue, The issue may not be resolved at all. 
```

Comment:
The comma after `article` probably shouldn't be there, and the `The` in the middle of the sentence shouldn't be capitalized.

Reply:
The "The" shouldn't be capitalized. That must have previously been two sentences. The commas act as parentheses.